impl/common/endpoints.go

```diff
@@ -38,6 +56,7 @@ func GetEndPoints(commandData *domain.CommandData, processRequest impl.RequestHe
 		commandData.ConnnectionData.LocatorAddress + "/management/",
 		commandData.ConnnectionData.LocatorAddress + "/management/v1/api-docs",
 		commandData.ConnnectionData.LocatorAddress + "/management/experimental/api-docs",
+		commandData.ConnnectionData.LocatorAddress + "/management/v3/api-docs",
```

Comment:
if these are tried in order, as an optimization maybe the most likely ones should be tried first? e.g. v3, then v1, then the others